lisa/tools/hwclock.py

```diff
@@ -62,3 +66,20 @@ def get(self, no_error_log: bool = True) -> datetime:
                 f" error: {command_result.stderr}"
             )
         return parser().parse(command_result.stdout)
+
+class HwclockFreebsd(Hwclock):
+
+    @property
+    def command(self) -> str:
+        return ""
+
+    @property
+    def can_install(self) -> bool:
+        return False
+
+    def _check_exists(self) -> bool:
+        return True
+
+    def set_rtc_clock_to_system_time(self) -> None:
+        cmd_result = self.node.execute("adjkerntz -i")
+        cmd_result.assert_exit_code()
```

Comment:
The error code assertion can be in same call with `execute`. Check its parameters.

Reply:
I updated the command to have the expected exit code in the execute line. I was previously just following the format of the existing function in Hwclock which has the exit value assertion on a separate line. Should I change that one to match as well?

Reply:
You don't need to clean up existing code, there are a lot of place...